src/transformers/models/llama/modeling_llama.py

```diff
@@ -41,6 +41,49 @@
 from .configuration_llama import LlamaConfig
 
 
+class AttentionMask2DTo4D:
```

Comment:
Do we plan to move this to the modelling utils or is this gonna be here for all models?

Reply:
Either #Copied from or we move it to a utils file. Both would work for me